frontend/screens/conversationHistory/components/ListItem.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { ConversationHistoryParametersInterface } from '../interfaces/ConversationHistoryParameters.interface';
+
+interface IProps {
+  query: string,
+  answer: string,
+  intent: string,
+  parameters: Array<ConversationHistoryParametersInterface>,
+  timestamp: Date,
+}
+
+/**
+ * This class implements the conversationHistory ListItem and its functionality
+ * @class ListItem
+ */
+export default class ListItem extends Component<IProps> {
+
+  public render() {
+
+    let parameters: string = '';
+
+    if (this.props.parameters.length === 0 ) {
+
+      parameters = 'Keine Parameter erkannt';
+
+    }
+
+    for (let i = 0; i < this.props.parameters.length; i++) {
```

Comment:
Einfacher ist oft die Verwendung von forEach():

`this.props.parameters.forEach((parameter, i) => {
 // Code here
})`

Reply:
Hast natürlich recht. Das Endprodukt hier ist auf einem etwas verqueren Weg entstanden. Ändere ich wahrscheinlich noch.